<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_okazanie_uslug</class>
 <widget class="QFrame" name="frm_okazanie_uslug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>631</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLineEdit" name="number">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>12</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>2</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>15</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Номер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>52</y>
     <width>46</width>
     <height>11</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Клиент:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>96</y>
     <width>46</width>
     <height>11</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Мастер:</string>
   </property>
  </widget>
  <widget class="QTableView" name="USLUGI">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>170</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>19</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QSelLineEdit" name="Client">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSelLineEdit" name="Sotrudnik">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="Materials">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>290</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>19</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="prodaja">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>410</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>19</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="groupMat">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>263</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="add_material">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/7.ico</normaloff>:/res/icons/7.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="del_material">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/5.ico</normaloff>:/res/icons/5.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupUsl">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>143</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolButton" name="add_usluga">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/7.ico</normaloff>:/res/icons/7.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="del_usluga">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/5.ico</normaloff>:/res/icons/5.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupMag">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>383</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QToolButton" name="add_prodaja">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/7.ico</normaloff>:/res/icons/7.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="del_prodaja">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/5.ico</normaloff>:/res/icons/5.ico</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Usl">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>143</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Услуги</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mat">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>263</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Материалы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mag">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>384</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Магазин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>12</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Date">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>12</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupOplati">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>504</y>
     <width>325</width>
     <height>109</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="Skidka">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>76</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="inputMask">
     <string>999; </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>4</y>
      <width>99</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Способ оплаты:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSkidka">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>77</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Продать со скидкой</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="spisanie_so_scheta">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>52</y>
      <width>157</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Списать остатки со счета</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sposobOplati">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>24</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="but_oplatit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>24</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Оплатить</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/res/icons/27.ico</normaloff>:/res/icons/27.ico</iconset>
    </property>
    <property name="autoRepeatDelay">
     <number>303</number>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSelLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qsellineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
